rdflib/plugins/stores/sparqlstore.py

```diff
@@ -13,67 +13,20 @@
 
 import re
 import collections
-import warnings
-import contextlib
 
-try:
-    from SPARQLWrapper import SPARQLWrapper, XML, JSON, POST, GET, URLENCODED, POSTDIRECTLY
-except ImportError:
-    raise Exception(
-        "SPARQLWrapper not found! SPARQL Store will not work." +
-        "Install with 'pip install SPARQLWrapper'")
+from .sparqlwrapper import SPARQLWrapper
```

Comment:
can we somehow avoid using SPARQLWrapper as a name (any caps)? Maybe SPARQLConnector? Or not expose it in its own module and internalize it here as `_SPARQLWrapper`?

I really would like to avoid the confusion